### DISCUSSION TITLE
Adopting ansible-lint-junit inside the linter

### DISCUSSION THREAD

**Discussion by webknjaz** [ansible/ansible-lint#396] [Ideas]
Hi, I've found this project https://github.com/wasilak/ansible-lint-junit, which adds junit output support.
I've checked the source and it's just ~60LOC. So it's fairly easy to incorporate it into the upstream project.

Among benefits, it'd facilitate dependent projects like CI integrations, ansible-review and probably even Galaxy.

This duplicates #265, but I thought I'd bring it up in the top of issue list :)

# Issue Type
- Feature request

# Ansible and Ansible Lint details

N/A

# Desired Behaviour

Support emitting junit reports.

**Comment by gundalow** [comment]
@wasilak FYI

**Comment by ssbarnea** [comment]
If this is so small, it would be ok to add it. Happy to see a PR.

**Comment by webknjaz** [comment]
It'd be useful for things like https://github.com/marketplace/actions/junit-report-to-annotations.

**Comment by maunzCache** [comment]
Hm the referenced pip does not seem to work with ansible-lint 5.0.x . There is no output which creates a proper testsuite. Still this should be easy to implement by the reference if the parsing works.
